src/main/webapp/script.js

```diff
@@ -228,6 +234,7 @@ function formatURLs(url, parameters) {
 }
 
 function getTotalCosts(aggregated){
+  console.log(aggregated)
```

Comment:
I think you should erase this

Reply:
Good point! All comments have been removed in 93be99b.